src/dawn/engine_dawn.cc

```diff
@@ -1518,9 +1516,8 @@ Result EngineDawn::AttachBuffers(ComputePipelineInfo* compute_pipeline) {
     }
 
     if (buf_info.descriptor_set > kMaxDawnBindGroup - 1) {
-      return Result(
-          "AttachBuffers: Dawn has maximum of 4 bindGroups "
-          "(descriptor sets)");
+      return Result("AttachBuffers: Dawn has a maximum of " +
+                    std::to_string(kMaxDawnBindGroup) + "(descriptor sets)");
```

Comment:
nit: space before the ( in "(descriptor sets" otherwise you'll get `4(descriptor sets` instead of `4 (descriptor sets)` as the error.

Reply:
good eye!